core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/AggregationUtils.java

```diff
@@ -21,19 +21,18 @@
 /**
  * Common functions for aggregation related functionalities.
```

Comment:
The function(s) defined in this utility class seem to be unrelated to Aggregation.
They just so happen to be used in/from an aggregation plugin.

Maybe not a great name, but `NumericUtils`, `NumberUtils`, or `SchemaUtils` seems more appropriate.
What do you think?